packages/core/src/indexing/postgres/postgres-index-api.ts

```diff
@@ -54,8 +54,8 @@ export class PostgresIndexApi implements DatabaseIndexApi {
       tip: indexingArgs.tip.toString(),
       last_anchored_at: indexingArgs.lastAnchor,
       first_anchored_at: indexingArgs.firstAnchor,
-      created_at: indexingArgs.createdAt || this.dbConnection.fn.now(),
-      updated_at: indexingArgs.updatedAt || this.dbConnection.fn.now(),
+      created_at: indexingArgs.createdAt || new Date(), // we don't use this.dbConnection.fn.now(), because postgres datetime may have higher precision than js date; TODO: CDB-XXX: set postgres created_at and updated_at precision to 3
+      updated_at: indexingArgs.updatedAt || new Date(), // we don't use this.dbConnection.fn.now(), because postgres datetime may have higher precision than js date; TODO: CDB-XXX: set postgres created_at and updated_at precision to 3
```

Comment:
```suggestion
      updated_at: indexingArgs.updatedAt || new Date(), // we don't use this.dbConnection.fn.now(), because postgres datetime may have higher precision than js date; TODO: CDB-2006: set postgres created_at and updated_at precision to 3
```